docs/UserGuide.md

```diff
@@ -73,21 +73,22 @@ Format: `help`
 
 ### Listing address book records: `list`
 
-Shows a list of all existing properties, sellers, buyers, or clients in the address book.
+Shows a list of all existing sellers, buyers, clients (i.e. buyers and sellers), properties and meetings in the address book.
```

Comment:
Shows a list of all existing clients (i.e. buyers and sellers), properties, and meetings in the address book.